client/components/drop-zone/test/index.jsx

```diff
@@ -37,7 +24,9 @@ describe( 'DropZone', function() {
 	} );
 
 	after( function() {
-		delete window.MutationObserver;
+		if ( global.window && global.window.MutationObserver ) {
```

Comment:
ok, whew, i thought you _had_ to do it for some odd reason. 

I'd be wary of leaving stuff like this in there, just because someone coming across it later might think it was strictly necessary and spent a bunch of time and energy trying to figure out _why_ it was done. But maybe not. :smile: